addons/website/static/src/components/views/page_list.xml

```diff
@@ -41,7 +41,7 @@
             <i class="me-1 fa fa-globe"/><span t-esc="state.activeWebsite.name"/>
         </button>
         <div class="dropdown-menu">
-            <t t-foreach="websiteSelection" t-as="website" t-key="website">
+            <t t-foreach="websiteSelection" t-as="website" t-key="website.id">
```

Comment:
For tracking purpose, this was failing because (as per the doc):
> The t-key directive is useful for lists (t-foreach). A key should be a unique number or string (objects will not work: they will be cast to the "[object Object]" string, which is obviously not unique).